### ISSUE TITLE
GDQuest brand style/thought guide

### ISSUE THREAD

**Issue by razcore-rad** [GDquest/gdquest-projects#31]
As we go towards a more mature & professional look we should really put in some thought & work in a brand guide.

- to start off with here's a high level presentation: https://youtu.be/_oaEomdek-0, I couldn't find the guide they speak about though - vid. is from 2014.
- some more practical advice: https://youtu.be/jB57Pc1W6Ys - with some already known like logo, fonts etc., but it goes a bit more in depth.
- this one's a more customer-oriented "show", but it still has some nice points in it if you got time to go over it: https://youtu.be/mxgOY2Ms-YI. I need to rewatch it too.

We should have a document touching on most ideas:
- logo
- colors
- website (including well formatted template banners and any reusable graphical elements)
- fonts
- video intro/outro & general styleguide, including say transitions
- sounds (if needed - music style, fx?!)
- code quality & guide for: website & projects (godot, blender etc.)
- guidelines/guides & how to contribute to our projects. All of this I think can go in the brand styleguide
- text tone for sentences (related to guidelines). How to be friendly etc. Most of this is already in the community guidelines, but having it in the brand styleguide I think makes sense - maybe a shorter version/main points?
- photos from events
- screenshots?! (border, cropping style, sturation etc.)
- quality guide
- UI/UX for products
- how to use GitHub (it is our main hub afterall): think issues, projects, CI (if applicable) etc.
- Kickstarter/Mavenseed/Gumroad/any-other-electronic-service layout/presentation
- discord/tweeter/community style (by which I mean have templates for: announcements, screenshot presentations, tweets that are related to specific topics/products?! etc.)

Any other stuff we can think of.

**Comment by NathanLovato** [comment]
I already have a brand document, and a good idea of what gdquest is and where it's headed.and I don't feel like we're having problems that'd justify investing work in that now.

What I see in that list is a wish to define and maybe standardize everything we do. With one guy full-time and a few part-time, it's really not a priority for me. I don't see this increasing our productivity now, helping us grow a bigger community - with the small size of gdquest, we need content for that -, or helping us increase revenue, which is what we need now. 

A few points:

- The list above isn't all about branding. Marketing/communication and code style, for instance, are completely unrelated.
- I already have a "brand document" and put thoughts in the project. 
- The colors, fonts, UI/UX for products etc. will be different per project. We're not google, creating our apps and services from scratch. For instance, if we make Godot plugins, the UX depends of Godot. For Blender, it's based on Blender. I don't see a project except for the website where we need to produce our own UX guidelines right now.
- There's no need for guidelines for screenshots, tones etc. 

Contribution guides etc. have nothing to do with a styleguide.

**Comment by NathanLovato** [comment]
First, what you have there it's a mixed bag. It's not just about brand design. I'll only talk about a brand styleguide below.

I already have a brand document, and a good idea of what gdquest is and where it's headed. Our visuals are already pretty consistent now, so is the communication, that I still mostly do myself, and the color palette is kind of there.

What are the problems that we have now that would require defining and standardizing everything we do? What are the concrete benefits of coming up with guidelines for everything listed above?

In a company with dozens or hundreds of employees, I get the purpose of a strong styleguide. For instance, google creates apps from scratch, so it's useful for them to have a common visual language for all their teams to produce similar output.

But for us, producing plugins, that rely on the host program's UX, with 1 guy full-time and 3 working at most several hundred hours a year, I don't see it as a priority.

Right now, I'm interested in work that can increase our productivity, help us grow a bigger community or that will help increase our revenue. And at our scale that happens when we create new content and spread the word.

**Comment by NathanLovato** [comment]
I'm okay with the idea of progressively building guidelines, and that's what we've been doing already. We can start to put together some document regarding the graphic design and communication style for GDQuest. 

I want us to build a style guide for the game demos that we make for example. That would certainly be useful moving forward with making projects in parallel.

But I'd like us to keep doing that as we go, as these evolve with our needs and scale.

**Comment by razcore-rad** [comment]
That's exactly what I thought you'd say and I agree. This isn't a priority and it's not for working on a document right now as an individual project, but more as a gradual process. Like you said, we already started building guidelines for community, for other projects etc. 

I got to this idea because of a few things we previously discussed and how we worked up to now. So you mentioned you want to up the game and have a more professional look and feel, starting with the main website as the hub. For me this implies consistency. And I agree that most of the stuff is consistent, but different things start to reveal, like the lack of naming conventions for the repos giving a disorganized feel to the (GitHub) organization and so on. This, and the fact that we already discussed, as you mentioned, having a unified visual language for the Godot demos made me think about this document.

Tone and sounds and whatever else I added in that list might not be part of the brand per se, but I actually think that this would be quite a nice help for anyone starting off a collaboration with you, including us working on different parts of the projects.

For example. It would have been very useful to me to know the tone we want to have when I worked on the "Learn Godot" documents. Some well placed examples with a bit of an explanation would have been pretty nice. Now I'm not saying I would have done a better job just cause I'd have this document, but I think it would have been a nice starting place.

Same with making the vids. I made really one tutorial, but it would have been nice to know what and how to do some of the things. For example, initially I was going to use an arrow, or highlight text with boxes and so on. I think we scratched that idea, right? Can't remember at this hour what the actual content is. Anyway, point is, would have been really nice to have a direction, which can go into this branding document - for internal use. Some links to some assets if need. How to do transitions etc. - possibly even define the exact type so all videos of a certain category (tutorials - in this case) use it?

And so on. I can see a lot of uses. Johnny used the coding styleguide for his work on remaking the godot platformer demo. This is the stuff I can see happening but on broader projects - starting places or guides that lay out basic requirements, easy to follow instructions to keep stuff standardized & consistent.

You mentioned you want us to start doing more communication. This all stuff that I started thinking after we had these conversations. I think it would be a great document to on-board other collaborators as well. Like artists that you want to hire and so on.

So just to reiterate, I completely agree with you that this isn't a priority. It's something I'm sure we'll build with time and the list I made is just a dump of everything I could think of, it's not necessarily the direction this document should take. It's just to start the conversation and see what makes sense and what doesn't.

**Comment by NathanLovato** [comment]
Sure it can be useful, also just writing a doc pushes you to better think about anything's design, to increase the coherence of your content or your understanding of your own work.

**Comment by razcore-rad** [comment]
So could you maybe share this guide with us? I think we should make a list and go over some of the issues we used in our work just to have a reference for later use. So I'm thinking about stuff like when you explained what our workflow should be when doing the fake game jams or when you explained stuff like tone to have in our posts (learn godot docs.) as there's already some great examples that we could use.

These might not be part of the "brand", but I think there's a case to be made that they should go into the document? Or at least link to these documents. The reason why I think it's a good idea to have it all under the brand styleguide is that we'd have a central document where we can consult all of this stuff instead of having it spread out over multiple documents. So that's why I think that having at least a TOC sort of thing with links to other documents, even guidelines, is a good idea to have in the brand styleguide. We could call it something else - brand guidelines which is more generic. I just used styleguide here cause the first thought I had was about design & look.

**Comment by NathanLovato** [comment]
The issue and the list above is a mixed bag of unrelated points. For instance:

> I'm thinking about stuff like when you explained what our workflow should be when doing the fake game jams

Game creation workflow training. This has nothing to do with GDQuest or us specifically, it could be a tutorial on the channel.

> The reason why I think it's a good idea to have it all under the brand styleguide is that we'd have a central document where we can consult all of this stuff instead of having it spread out over multiple documents

That's the contrary for me, different persons need different documents. If I've learned one thing so far, it's that it's hard to get people to read documents or to pay attention. And this year's taught me that this is also true for the people you pay. 

So for me, to be productive, our guides should:

1. Solve the problems we have. Once a need arises on a given project, I can spend a few hours writing a doc.
2. Be focused. The shorter and the clearer a resource is, the easier it is to commit info to memory.

Anyway, there's already a task to add the learn theme to the website and to use it for our own docs and guidelines, there are already guidelines, and we're already in the process of adding more as we go.
